libsql-client/examples/select.rs

```diff
@@ -0,0 +1,76 @@
+use libsql_client::{Connection, QueryResult, Statement, Value};
+use rand::prelude::SliceRandom;
+
+fn result_to_string(result: QueryResult) -> String {
+    let mut ret = String::new();
+    match result {
+        QueryResult::Error((msg, _)) => return format!("Error: {msg}"),
+        QueryResult::Success((result, _)) => {
+            for column in &result.columns {
+                ret += &format!("| {column:16} |");
+            }
+            ret += "\n| -------------------------------------------------------- |\n";
+            for row in result.rows {
+                for column in &result.columns {
+                    ret += &format!("| {:16} |", row.cells[column].to_string());
+                }
+                ret += "\n";
+            }
+        }
+    };
+    ret
+}
+
+// Serve a request to load the page
+async fn serve(db: impl Connection) -> String {
+    // Recreate the tables if they do not exist yet
+    db.execute("CREATE TABLE IF NOT EXISTS counter(country TEXT, city TEXT, value, PRIMARY KEY(country, city)) WITHOUT ROWID").await.ok();
+    db.execute(
+        "CREATE TABLE IF NOT EXISTS coordinates(lat INT, long INT, airport TEXT, PRIMARY KEY (lat, long))",
+    ).await.ok();
```

Comment:
Just an idea. Maybe it's worth using `batch` here?

Reply:
technically yes, but I wanted to show both `execute()` and `transaction()` in this example, so I split this one :sweat_smile:

Reply:
I think select below uses execute :).

Reply:
indeed, now the example is way better, because it covers execute, batch *and* transaction. thx!